CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Support for STAC Catalogs and link objects. [#9](https://github.com/stac-extensions/file/pull/9)
```

Comment:
We can move this to v2.1.0 as it has not been released yet.

Reply:
Done/pushed.